packages/app-check/src/index.ts

```diff
@@ -46,6 +47,10 @@ function registerAppCheck(firebase: _FirebaseNamespace): void {
       },
       ComponentType.PUBLIC
     )
+      .setServiceProps({
```

Comment:
Need to add them to the `declare module` block at the end of the file.

```
  interface FirebaseNamespace {
    appCheck(app?: FirebaseApp): FirebaseAppCheck;
    ReCaptchaV3Provider: typeof ReCAPTCHAV3Provider;
    CustomProvider: typeof CustomProvider;
  }
```

Reply:
Done.